gnmlibrarytool/urls.py

```diff
@@ -30,4 +31,12 @@
     url(r'^diagram$', login_required(DiagramMainView.as_view())),
     url(r'^endpoint/', include(router.urls)),
     url(r'^storageruleinfo/(\w{2}-\d+)$',StorageRuleInfoView.as_view(), name="libtool_storage_info_view"),
+    url(r'^rules/$', login_required(rule_form), name='rules'),
+    url(r'^add/$', login_required(add_rule)),
+    url(r'^addrule/$', login_required(add_rule_to_item)),
+    url(r'^delete/$', login_required(delete_rule)),
```

Comment:
I would prefer cleaner naming here. Something like ^itemrule/(?<itemid>\w{2}-\d+)/add and delete for item override views and ^rule/update/(?<ruleid\d+) etc. for defining views in the models. It is not clear what the difference between add and addrule is.